build failed on MacOS, version 12.0.1

Platform: Mac OS
version: 12.0.1 
reason: build failed

Warning: Mapping new ns http://schemas.android.com/repository/android/generic/02 to old ns http://schemas.android.com/repository/android/generic/01
Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/addon2/02 to old ns http://schemas.android.com/sdk/android/repo/addon2/01
Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/repository2/02 to old ns http://schemas.android.com/sdk/android/repo/repository2/01
Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/sys-img2/02 to old ns http://schemas.android.com/sdk/android/repo/sys-img2/01

> Task :app:processDebugManifest
/Users/matthew/Downloads/github/FudiNFC/app/src/main/AndroidManifest.xml:15:5-40:19 Warning:
        application@android:label was tagged at AndroidManifest.xml:15 to replace other declarations but no other declaration present

> Task :app:processDebugResources FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processDebugResources'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
   > Android resource linking failed
     /Users/matthew/.gradle/caches/transforms-2/files-2.1/222fb1e9b4241bd7b141674dfe2701ed/core-1.8.0-alpha02/res/values/values.xml:105:5-114:25: AAPT: error: resource android:attr/lStar not found.


* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 11s

the build failed, it's seem that you use the "+" for the version of androidx.core:core-ktx, it should be 1.6.0.

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -45,7 +45,7 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.0.0'
     implementation 'com.android.support:design:29.0.0'
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation "androidx.core:core-ktx:+"
+    implementation "androidx.core:core-ktx:1.6.0"
     implementation 'com.airbnb.android:lottie:3.7.0'
 
     implementation "org.koin:koin-android:2.0.0"

--- a/fudi-nfc/build.gradle
+++ b/fudi-nfc/build.gradle
@@ -46,7 +46,7 @@ android {
 dependencies {
     implementation 'androidx.appcompat:appcompat:1.0.0'
 //    implementation 'com.intellij:annotations:12.0'
-    implementation "androidx.core:core-ktx:+"
+    implementation "androidx.core:core-ktx:1.6.0"
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
 }
 

